protos/dcs/common/v0/common.proto

```diff
@@ -171,6 +171,10 @@ message Position {
   double lon = 2;
   // Altitude in Meters above Mean Sea Level (MSL)
   double alt = 3;
+  // Distance between DCS' map origin to object in meters on west-east axis.
+  double u = 4;
+  // Distance between DCS' map origin to object in meters on north-south axis.
+  double v = 5;
```

Comment:
This adds `u`/`v` to all positions. I'd be up for changing the implementation to only include it for the position of units (might also be useful in the future for projectiles in case anyone wants to export them to TacView).

Reply:
👍